jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,113 @@
+
+* TOC 
+{:toc}
+
+## Introduction to API v2
+
+CircleCI API v2 has significantly changed the current API v1.1 paradigm to incorporate some important changes and shifts in the way that you can and will use the API for your jobs. Although changes were made to the available endpoints (discussed later on this page), two of the most important changes are the introduction of "pipelines" and "project slugs" as concepts in the API design and architecture, as well as a set of new endpoints that enable you to:
+
+- Retrieve a workflow by ID
+- Trigger pipelines with parameters
+- Trigger particular workflows within pipelines
+- Retrieve a pipeline by ID
+- Retrieve recent pipelines for an organization or for a project (ideally with branch filtering)
+
+**Note** In API v1.1, the term "build processing" was used to describe how projects were built and processed; however, with the API v2 release, "build processing" has been replaced by "pipelines." Not only has the terminology changed, but all projects have been moved over to this new setting by default.
+
+### Authentication
+
+The CircleCI API v2 enables users to be authenticated in much the same way as they would be authenticated in the previous API v1.1. A simple way to authenticate is to send your API token as the username of the HTTP request. For example, if you have set `CIRCLECI_TOKEN` in your shell's environment, then you could then use `curl` with that token like the examnple shown below:
+
+`curl -u ${CIRCLECI_TOKEN}: https://circleci.com/api/v2/me`
+
+For more detailed information on how to authenticate with a token, please refer to the [Authentication](https://circleci.com/docs/api/#authentication) section in the main CirlceCI API documentation.
+
+### Pipelines
+
+#### What is a Pipeline?
+
+"Pipelines" are a CircleCI designation for the full set of configurations you run when triggering work on your projects on the CircleCI platform. Pipelines contain your workflows, which in turn coordinate your jobs. 
+
+#### What Are the Practical Implications of Pipelines?
+
+In this new paradigm, all jobs will run in a workflow, even if you do not specify one in your configuration. In the future, CircleCI will add a new pipeline-level parameters clause in `config.yml`, referenced in the `pipeline` scope, as well as launch a new version of the CircleCI API that has the ability to trigger pipelines with parameters and retrieve them and their workflows. In addition to these changes, CircleCI is also creating a new User Interface that will unify the experience of viewing your pipelines and their jobs coordinated in workflows.
+
+#### What Does This Change for Me? How Do I Turn On Pipelines?
+
+Whatever you are running today should continue working without any intervention. CircleCI has some features (such as 2.1 configuration and orbs) that require you to enable pipelines. In some rare cases, some users may need to make tweaks to their configuration when enabling pipelines.
```

Comment:
This is one for @ndintenfass, but how do we capitalise pipelines? In all documents I've written so far I've capitalised everything that's a product term because it feels like a proper noun.